src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_lord_jaraxxus.cpp

```diff
@@ -111,592 +108,459 @@ enum Misc
-                            break;
-                        case EVENT_SUMMON_INFERNAL_ERUPTION:
-                            Talk(EMOTE_INFERNAL_ERUPTION);
-                            Talk(SAY_INFERNAL_ERUPTION);
-                            DoCast(SPELL_INFERNAL_ERUPTION);
-                            events.ScheduleEvent(EVENT_SUMMON_INFERNAL_ERUPTION, 2*MINUTE*IN_MILLISECONDS);
-                            break;
-                        case EVENT_INTRO:
-                            DoCastSelf(SPELL_LORD_JARAXXUS_HITTIN_YA, true);
-                            me->GetMotionMaster()->MoveAlongSplineChain(POINT_SUMMONED, SPLINE_INITIAL_MOVEMENT, true);
-                            break;
-                        case EVENT_TAUNT_GNOME:
-                            Talk(SAY_INTRO);
-                            events.ScheduleEvent(EVENT_KILL_GNOME, 9s);
-                            break;
-                        case EVENT_KILL_GNOME:
-                            DoCastSelf(SPELL_FEL_LIGHTNING);
-                            events.ScheduleEvent(EVENT_CHANGE_ORIENTATION, 3s);
-                            break;
-                        case EVENT_CHANGE_ORIENTATION:
-                            me->SetFacingTo(4.729842f);
-                            events.ScheduleEvent(EVENT_START_COMBAT, 7s);
-                            break;
-                        case EVENT_START_COMBAT:
-                            me->SetImmuneToPC(false);
-                            me->SetReactState(REACT_AGGRESSIVE);
-                            DoZoneInCombat();
-                            break;
-                        default:
-                            break;
+                        Talk(EMOTE_INCINERATE, target);
+                        Talk(SAY_INCINERATE);
+                        DoCast(target, SPELL_INCINERATE_FLESH);
                     }
-
-                    if (me->HasUnitState(UNIT_STATE_CASTING))
-                        return;
+                    events.Repeat(23s);
+                    break;
+                case EVENT_NETHER_POWER:
+                {
+                    CastSpellExtraArgs args(TRIGGERED_FULL_MASK);
+                    args.SpellValueOverrides.AddMod(SPELLVALUE_AURA_STACK, RAID_MODE(5, 10, 5, 10));
+                    me->CastSpell(me, SPELL_NETHER_POWER, args);
+                    events.Repeat(42s);
+                    break;
                 }
-
-                DoMeleeAttackIfReady();
+                case EVENT_LEGION_FLAME:
+                    if (Unit* target = SelectTarget(SELECT_TARGET_RANDOM, 1, 0.0f, true))
+                    {
+                        Talk(EMOTE_LEGION_FLAME, target);
+                        DoCast(target, SPELL_LEGION_FLAME);
+                    }
+                    events.Repeat(30s);
+                    break;
+                case EVENT_SUMMON_NETHER_PORTAL:
+                    Talk(EMOTE_NETHER_PORTAL);
+                    Talk(SAY_MISTRESS_OF_PAIN);
```

Comment:
is this talks without delays?

Reply:
Looks like it. I see no problem with no delay between an emote line and a "say" line from the same creature.

Reply:
yes, is 1 emote and 1 say :)